include/seqan3/core/platform.hpp

```diff
@@ -22,6 +22,14 @@
 #define SEQAN3_STR(x) SEQAN3_STR_HELPER(x)
 //!\endcond
 
+// ============================================================================
+//  Compiler support
+// ============================================================================
+
+#if defined(__GNUC__) && (__GNUC__ == 7 || __GNUC__ == 8)
```

Comment:
Maybe make it `< 9` ? So the warning also is emitted for acc < 7. 
Or do you want to have a different warning for < 7?